catalogue_api/api/src/main/scala/uk/ac/wellcome/platform/api/requests/Requests.scala

```diff
@@ -19,6 +19,7 @@ trait MultipleResultsRequest[W <: WorksIncludes] extends ApiRequest {
   val include: Option[W]
   val query: Option[String]
   val workType: Option[String]
+  val itemLocationType: Option[String]
```

Comment:
if workType and locationType are not available for V1 resquests why do we add them here instead of V2MultipleResultsRequest?

Reply:
Because the logic for building the filters is inside WorksController, and is quite generic about the request type.

I want to keep it generic because I think some of the filters will get backported to V1 (contributors and subjects), and it leaves the door open for generic filters in V3. But not strongly tied to it.

Reply:
Just to confirm, there will be no requirement for filtering in V1.